### DISCUSSION TITLE
CAST : Add support to cast on DLNA devices

### DISCUSSION THREAD

**Discussion by MrSiO** [SamTV12345/PodFetch#762] [Ideas]
Here is a suggestion:   Add the possibility to CAST to DLNA devices on network. 

Great exemple and integration of this would be "Jellyfin":    https://github.com/jellyfin/jellyfin

Thx for considering!

**Comment by SamTV12345** [comment]
> Here is a suggestion: Add the possibility to CAST to DLNA devices on network.
> 
> Great exemple and integration of this would be "Jellyfin": https://github.com/jellyfin/jellyfin
> 
> Thx for considering!

Good idea. Is there some kind of API documentation and maybe a test setup how can check if it really does work?

**Reply by MrSiO** [reply]
I'm no coder but the project is open source so you could probably get some info from the GuiHub.
